packages/wdio-browserstack-service/src/instrumentation/funnelInstrumentation.ts

```diff
@@ -0,0 +1,142 @@
+import * as os from 'node:os'
+import * as util from 'node:util'
+import path from 'node:path'
+import fs from 'node:fs'
+import got from 'got'
+import UsageStats from '../testOps/usageStats.js'
+import { BStackLogger } from '../bstackLogger.js'
+import type BrowserStackConfig from '../config.js'
+import { BSTACK_SERVICE_VERSION, FUNNEL_INSTRUMENTATION_URL } from '../constants.js'
+import DataStore from '../data-store.js'
+
+class FunnelTestEvent {
+    static async fireFunnelTestEvent(eventType: string, config: BrowserStackConfig) {
+        try {
+            if (!config.userName || !config.accessKey) {
+                BStackLogger.debug('username/accesskey not passed')
+                return
+            }
```

Comment:
This can be moved outside of the `try/catch` block

Reply:
done